.github/workflows/quality-checks.yml

```diff
@@ -12,8 +12,28 @@ jobs:
       - run: npm install
       - run: npm run format:check
       - run: npm run lint
-      - run: npm run test
-      - run: npm run test:it
+      # Disabled for draft purpose - delete before merge
+      #      - run: npm run test
+      #      - run: npm run test:it
+      - name: Set environment
+        run: |
+          sudo echo "127.0.0.1 prod.foo.redhat.com" | sudo tee -a /etc/hosts
+          TOKEN=${{ secrets.CYPRESS_OPENSHIFT_OFFLINE_TOKEN }} ./scripts/get-sso-token.sh >> $GITHUB_ENV
+      - name: E2E tests
+        uses: cypress-io/github-action@v4
+        with:
+          install: false
+          project: ./cypress
+          start: npm run start
+          wait-on: "https://prod.foo.redhat.com:1337"
+          wait-on-timeout: 120
+          config-file: cypress.e2e.json
+        env:
+          NODE_TLS_REJECT_UNAUTHORIZED: 0
+          CYPRESS_USER: ${{ secrets.CYPRESS_USER }}
+          CYPRESS_PASSWORD: ${{ secrets.CYPRESS_PASSWORD }}
+          CYPRESS_SANDBOX_DEV_REST_URL: ${{ secrets.CYPRESS_SANDBOX_DEV_REST_URL }}
+          CYPRESS_SANDBOX_DEV_REST_PATH: ${{ secrets.CYPRESS_SANDBOX_DEV_REST_PATH  }}
```

Comment:
@jomarko can you check if these secrets are without typos? I have tried the `cypress-io/github-action@v4`. Based on output:
- the `start` is performed
- the service is listening
- the cypress is performed  (the E2E test does not load ENV)

I get this error:

![image](https://user-images.githubusercontent.com/1536609/179538090-316c9a7c-e0d0-404f-9108-31991edb4828.png)